google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/TableAdminRequests.java

```diff
@@ -0,0 +1,236 @@
+      tableRequest.setGranularity(granularity);
+      return this;
+    }
+
+    /**
+     * Adds a new columnFamily to the configuration
+     *
+     * @param familyId
+     */
+    public CreateTable addColumnFamily(String familyId) {
+      Preconditions.checkNotNull(familyId);
+      tableRequest.putColumnFamilies(familyId, ColumnFamily.newBuilder().build());
+      return this;
+    }
+
+    /**
+     * Adds a new columnFamily with {@link GCRule} to the configuration
+     *
+     * @param familyId
+     * @param gcRule
+     */
+    public CreateTable addColumnFamily(String familyId, GCRule gcRule) {
+      Preconditions.checkNotNull(familyId);
+      tableRequest.putColumnFamilies(
+          familyId, ColumnFamily.newBuilder().setGcRule(gcRule.toProto()).build());
+      return this;
+    }
+
+    /**
+     * Adds split at the specified key to the configuration
+     *
+     * @param key
+     */
+    public CreateTable addSplit(ByteString key) {
+      Preconditions.checkNotNull(key);
+      createTableRequest.addInitialSplits(Split.newBuilder().setKey(key).build());
+      return this;
+    }
+
+    /**
+     * Adds the specified number of uniform splits to the configuration
+     *
+     * @param key
+     * @param numSplits
+     */
+    public CreateTable addUniformSplits(ByteString key, int numSplits) {
+      Preconditions.checkNotNull(key);
+      // TODO: add implementation
+      throw new UnsupportedOperationException();
+    }
+
+    @InternalApi
+    public CreateTableRequest toProto(InstanceName instanceName) {
+      Preconditions.checkNotNull(instanceName);
+      return createTableRequest
+          .setParent(instanceName.toString())
+          .setTable(tableRequest.build())
+          .build();
+    }
+  }
+
+  /**
+   * Fluent wrapper for {@link ModifyColumnFamiliesRequest}
+   *
+   * <pre>
+   * Allows the following ColumnFamily modifications
+   *    - create family, optionally with {@link GCRule}
+   *    - update existing family {@link GCRule}
+   *    - drop an existing family
+   * </pre>
+   */
+  public static final class ModifyFamilies {
+    private final ModifyColumnFamiliesRequest.Builder modFamilyRequest =
+        ModifyColumnFamiliesRequest.newBuilder();
+    private String tableId;
+
+    /**
+     * Configures the tableId to execute the modifications
+     *
+     * @param tableId
+     */
+    private ModifyFamilies(String tableId) {
+      Preconditions.checkNotNull(tableId);
+      this.tableId = tableId;
+    }
+
+    /**
+     * Configures the name of the new ColumnFamily to be created
+     *
+     * @param familyId
+     * @return
+     */
+    public ModifyFamilies create(String familyId) {
+      return createWithGCRule(familyId, null);
+    }
+
+    /**
+     * Configures the name and GcRule of the new ColumnFamily to be created
+     *
+     * @param familyId
+     * @param gcRule
+     * @return
+     */
+    public ModifyFamilies createWithGCRule(String familyId, GCRule gcRule) {
```

Comment:
please be consistent, you were using the verb add above now you are using create. Also I would drop WithGcRule suffix